Amplify/Core/Support/DeviceInfo.swift

```diff
@@ -18,7 +18,9 @@ import AppKit
 #endif
 
 public struct DeviceInfo {
```

Comment:
Is this a new struct? Does it need to be in Amplify Core?

Reply:
It needs to be in Amplify Core because Analytics and Auth, both depend on it.